crates/sui-indexer-alt/Cargo.toml

```diff
@@ -42,8 +42,13 @@ sui-pg-temp-db.workspace = true
 sui-protocol-config.workspace = true
 sui-storage.workspace = true
 sui-types.workspace = true
+sui-synthetic-ingestion = { workspace = true, optional = true }
 
 [dev-dependencies]
 rand.workspace = true
 tempfile.workspace = true
 wiremock.workspace = true
+
+[features]
+default = []
+benchmark = ["sui-synthetic-ingestion"]  # This will be used to enable benchmark mode
```

Comment:
This is mostly for my own education -- in some places, I've seen this written like it is here, and in other cases the feature name is prefixed with a "dep:" when it enables the dependency.

Is there a difference between the two?

Reply:
Good question. I am not confused on the difference too.